src/lib/patchConceptForm.ts

```diff
@@ -15,10 +15,18 @@ export const patchConceptFromForm = (
   { concept, dispatch, lastPatchedResponse = {}, isSaving }
 ): void => {
   const diff = compare(
-    omit(lastPatchedResponse, 'endringslogelement'),
-    omit({ ...lastPatchedResponse, ...values }, 'endringslogelement')
+    omit(lastPatchedResponse, [
```

Comment:
Jeg skjønte ikke helt hvorfor det gjøres her..

Reply:
For at man skal kunne slette et felt må det være til stede i `lastPatchedResponse` og ikke i `values`. Når vi spreader begge sammen vil vi få oppdaterte verdier satt riktig, men om en verdi mangler i values vil den da fortsatt ha den fra lastPatchedResponse